Scrollable/Scrollable.js

```diff
@@ -340,7 +340,10 @@ class ScrollableBase extends Component {
 		this.stopOverscrollJob('vertical', 'after');
 	}
 
+	flickTarget = null
```

Comment:
We don't appear to be using `flickTarget` other than as a flag that the user flicked. If I'm reading that right, let's rename it here and remove it from the change to `enact`. I'd like to avoid decorating events with additional members so the payloads are consistent. If we need new data, we should create new events.

Reply:
Ah, you're right. An earlier iteration actually used the values returned in the event, but I don't believe we need them now.